Gems/Atom/Tools/MaterialCanvas/Code/Source/Document/MaterialGraphCompiler.cpp

```diff
@@ -320,14 +326,17 @@ namespace MaterialCanvas
 
     void MaterialGraphCompiler::ClearFingerprintsForCurrentNode()
     {
-        for (const auto& templatePath : m_templatePathsForCurrentNode)
+        if (AtomToolsFramework::GetSettingsValue("/O3DE/Atom/MaterialCanvas/ForceClearAssetFingerprints", true))
```

Comment:
This key should really be a shared constant between where it's defined and where it's used.